src/main/kotlin/no/nav/syfo/fastlege/FastlegeService.kt

```diff
@@ -70,8 +68,7 @@ class FastlegeService @Inject constructor(
 
     private fun List<Fastlege>.aktiv(): Fastlege? {
         return this.filter { fastlege ->
-            val tomorrow = LocalDate.now().plusDays(1)
-            fastlege.gyldighet.fom.isBefore(tomorrow)
+            fastlege.relasjon.kodeVerdi == RelasjonKodeVerdi.FASTLEGE.kodeVerdi
```

Comment:
Burde vi ikke sjekke om fastlegen er aktiv på dato også? Eller vil den alltid være det, kanskje? 🤔

Reply:
Hmm, jeg håper jo at maks er 1 element i denne listen som har denne kodeVerdi'en, men er jo ikke garantert.
Den fastlege'en man har ikke en sluttdato, men tenker du da på når man evt. får en Fastlege som får en startdato som er i fremtiden?
Rente datamessig er jo dette mulig, men hadde ikke det vært rart om de gjør?

Reply:
Jeg tenkte mest på om det kan komme en gammel fastlege, men det kan vel egentlig ikke skje, nei. Og det vil jo synes i modia for veilederne uansett 👍